gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/alttext/AltTextPage.kt

```diff
@@ -175,7 +185,26 @@ internal fun AltTextPage(
                         textStyle = LocalTextStyle.current.copy(color = MaterialTheme.colorScheme.onSurface),
                         modifier = Modifier
                             .fillMaxWidth()
-                            .padding(start = 16.dp, top = 8.dp, bottom = 8.dp),
+                            .fillMaxHeight(),
+                        decorationBox = { innerTextField ->
+                            Column(
+                                modifier = Modifier
+                                    .padding(start = 16.dp, top = 8.dp, bottom = 8.dp),
+                            ) {
+                                Box(
+                                    modifier = Modifier.weight(1f),
+                                ) {
+                                    innerTextField()
+                                }
```

Comment:
The Box is used to take the available space via a weight modifier. I couldn't find a way to apply modifier to the inner text field.